### ISSUE TITLE
SINKIT_SINKHOLE_IP between Sinkit Core and GoDNS Resolver

### ISSUE THREAD

**Issue by Karm** [intfeed/sinkit-resolver#5]
Facts
---
Sinkit Core returns a sinkhole IP address as a JSON response to the GoDNS client, e.g. ```{"sinkhole":"127.0.0.1"}```. The GoDNS client unmarshalles the response, but instead of using it, it returns the IP address set in its ```SINKIT_SINKHOLE_IP``` env variable.

Problem
---
One could erroneously think that setting ```SINKIT_SINKHOLE_IP``` for Sinkit Core affects sinkhole ip address returned to the clients by GoDNS server. It is counter-intuitive that it is actually necessary to set ```SINKIT_SINKHOLE_IP``` env variable to the GoDNS server itself.

Solution
---
Decide whether it is Sinkit Core or GoDNS who sets the sinkhole IP address. I favour the Sinkit Core being the one who sets it, because according to its business logic, it could return various sink hole ip addresses to the GoDNS server.

@robcza Could you comment, please?

**Comment by robcza** [comment]
Yes, it has to be the core to have more control over it. Keep the resolver as simple as possible.
